docs/Changelog.md

```diff
@@ -34,16 +34,23 @@ sending a mail to <afl-users+subscribe@googlegroups.com>.
                the last 5 queue entries
       - rare: puts focus on queue entries that hits rare branches, also ignores
               runtime
-  - llvm_mode: added Control Flow Integrity sanitizer (AFL_USE_CFISAN)
+  - llvm_mode: 
+    - added SNAPSHOT feature (using https://github.com/AFLplusplus/AFL-Snapshot-LKM)
+    - added Control Flow Integrity sanitizer (AFL_USE_CFISAN)
+    - added AFL_LLVM_INSTRUMENT option to control the instrumentation type
+      easier: DEFAULT, CFG (INSTRIM), LTO, CTX, NGRAM-x (x=2-16)
+    - made USE_TRACE_PC compile obsolete
   - LTO collision free instrumented added in llvm_mode with afl-clang-lto -
     note that this mode is amazing, but quite some targets won't compile
   - Added llvm_mode NGRAM prev_loc coverage by Adrean Herrera
     (https://github.com/adrianherrera/afl-ngram-pass/), activate by setting
-    AFL_LLVM_NGRAM_SIZE
+    AFL_LLVM_INSTRUMENT=NGRAM-<value> or AFL_LLVM_NGRAM_SIZE=<value>
```

Comment:
Should we deprecate `AFL_LLVM_NGRAM_SIZE`?

Reply:
IMHO no. this is the original env var where I imported the code from. also we have AFL_LLVM_INSTRIM, AFL_LLVM_CTX, etc. and people are used to set these with =1.
I added the new AFL_LLVM_INSTRUMENT option to give an alternative and easier show that these are mutual exclusive option. Also makes it easier for the fuzzbench run :)